docs/api/doc-block-markdown.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Markdown'
+---
+
+The `Markdown` block allows you to import and include plain markdown in your MDX files.
+
+![Screenshot of Markdown block](TK)
+
+<!-- prettier-ignore-start -->
+```md
+// DON'T do this, will error
+import ReadMe from './README.md';
+// DO this, will work
+import ReadMe from './README.md?raw';
+
+import { Markdown } from '@storybook/blocks';
+
+# A header 
+
+<Markdown>{ReadMe}</Markdown />
+```
+<!-- prettier-ignore-end -->
```

Comment:
This paragraph from the drafts are omitted here (on purpose or not?), but I think it's an important companion to the snippet.

> When importing markdown files it’s important to use the ?raw suffix on the import path to ensure the content is imported as-is, instead of being evaluated:

```suggestion
The `Markdown` block allows you to import and include plain markdown in your MDX files.

![Screenshot of Markdown block](TK)

When importing markdown files it’s important to use the `?raw` suffix on the import path to ensure the content is imported as-is, instead of being evaluated:

<!-- prettier-ignore-start -->
```md
// DON'T do this, will error
import ReadMe from './README.md';
// DO this, will work
import ReadMe from './README.md?raw';

import { Markdown } from '@storybook/blocks';

# A header 

<Markdown>{ReadMe}</Markdown />
```
<!-- prettier-ignore-end -->
```

Reply:
Whoops! Thanks for the catch. Agreed it's important.